apps/docs/content/guides/functions/tests.mdx

```diff
@@ -0,0 +1,162 @@
+---
```

Comment:
We already have a guide for Testing here: https://docs-git-feat-functions-lw14-docs-updates-supabase.vercel.app/docs/guides/functions/unit-test

I think we should replace it with this post instead.